sdk/storage/azure-storage-file-datalake/tests/test_file.py

```diff
@@ -98,6 +98,22 @@ def test_create_file(self, datalake_storage_account_name, datalake_storage_accou
         # Assert
         self.assertIsNotNone(response)
 
+    @pytest.mark.live_test_only
```

Comment:
Why Live only here? You also generated recordings which would not be needed if this was Live only, but I think it can be recorded too? This applies for all tests.

Reply:
Good point- something tells me the reference test I was modeling from might have had that because of some SAS permissions that require live-testing only. Fixed!